### ISSUE TITLE
Downloading does not work

### ISSUE THREAD

**Issue by Dabbing-Guy** [sdaqo/anipy-cli#54]
**Describe the bug**
Anime downloader raises an assertation error when used.

**To Reproduce**
1. Run `anipy-cli -D`
2. Say no to kitsu (might not be necessary)
3. Search for your anime
4. Optionally add another another anime (does not matter)
5. Try to download it

**Expected behavior**
The anime gets downloaded

**Info**
```
Name: anipy-cli
Version: 2.4.13
Summary: Little tool in python to watch anime from the terminal (the better way to watch anime)
Home-page: https://github.com/sdaqo/anipy-cli
Author: sdaqo
Author-email: sdaqo.dev@protonmail.com
License: GPL-3.0
Location: /home/michael/.local/lib/python3.10/site-packages
Requires: better-ffmpeg-progress, bs4, kitsu.py-extended, m3u8, moviepy, pycryptodomex, pypresence, python-dateutil, requests, tqdm
Required-by: 
```

**Platform**
 - OS: Artix Linux
 - Python version 10.4
 - Latest version of anipy-cli

**Additional context**
Output
```***Download Mode***
Downloads are stored in: /home/michael/.local/lib/python3.10/site-packages/anipy_cli/download
Search Kitsu for anime in Season? (y|n): 
>> n   
Search: log horizon
Add another search: (y|n)
n

Current:  log horizon
[1] Log Horizon
[2] Log Horizon 2
[3] Log Horizon (Dub)
[4] Log Horizon: Entaku Houkai
[5] Log Horizon 2nd Season (Dub)
[6] Log Horizon: Entaku Houkai (Dub)
Enter Number: 1
Episode [1-25]: 16-25
--------------------
Downloading: Log Horizon EP: 16 - hlsp 
Type: m3u8
Downloader: internal
Selected Quality: hlsp
Playlist Index: 0
Resolution at this index: (1920, 1080)


Selected Quality: hlsp
Playlist Index: 1
Resolution at this index: (1280, 720)


Selected Quality: hlsp
Playlist Index: 2
Resolution at this index: (854, 480)


Selected Quality: hlsp
Playlist Index: 3
Resolution at this index: (640, 360)


Traceback (most recent call last):
  File "/home/michael/.local/bin/anipy-cli", line 8, in <module>
    sys.exit(main())
  File "/home/michael/.local/lib/python3.10/site-packages/anipy_cli/run_anipy_cli.py", line 8, in main
    cli.main()
  File "/home/michael/.local/lib/python3.10/site-packages/anipy_cli/cli.py", line 638, in main
    download_cli(args.quality, args.ffmpeg, args.no_kitsu)
  File "/home/michael/.local/lib/python3.10/site-packages/anipy_cli/cli.py", line 110, in download_cli
    download(show_entry, ffmpeg).download()
  File "/home/michael/.local/lib/python3.10/site-packages/anipy_cli/download.py", line 72, in download
    self.multithread_m3u8_dl()
  File "/home/michael/.local/lib/python3.10/site-packages/anipy_cli/download.py", line 247, in multithread_m3u8_dl
    assert self._m3u8_content.is_variant is False
AssertionError
                          ```

**Comment by sdaqo** [comment]
Thanks for reporting. I will take a look at this tomorrow, in the meantime you can use it with the ffmpeg flag.

**Comment by sdaqo** [comment]
Hey @dymattic, could you maybe look into this aswell, since you wrote it. I will take  a look as well, tough it would be nice if you could help out, thanks.

**Comment by dymattic** [comment]
> Hey @dymattic, could you maybe look into this aswell, since you wrote it. I will take  a look as well, tough it would be nice if you could help out, thanks.
> 
> 

Sure I'll take a look at it

**Comment by dymattic** [comment]
Fixed it #55
